subnet-private.tf

```diff
@@ -60,21 +60,10 @@ resource "aws_route" "nat_route" {
 resource "aws_route_table_association" "private" {
   count          = length(aws_subnet.private)
   subnet_id      = aws_subnet.private[count.index].id
-  route_table_id = var.multi_nat ? aws_route_table.private[count.index].id : aws_route_table.private[0].id
+  route_table_id = var.multi_nat  ? aws_route_table.private[0].id : aws_route_table.private[count.index].id
```

Comment:
This doesn't look right - when multi_nat is on, it has multiple NATs and multiple route tables for each private subnet